django/db/models/fields/related.py

```diff
@@ -1586,7 +1586,16 @@ class ForeignObject(RelatedField):
 
     def __init__(self, to, from_fields, to_fields, rel=None, related_name=None,
             related_query_name=None, limit_choices_to=None, parent_link=False,
-            on_delete=CASCADE, swappable=True, **kwargs):
+            on_delete=None, swappable=True, **kwargs):
+
+        if on_delete is None:
+            warnings.warn(
+                "on_delete will be a required kwarg for %s in Django 2.0. "
+                "Set the argument explicitly instead. "
+                "See https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.ForeignKey.on_delete" % self.__class__.__name__,
```

Comment:
use `django.utils.version.get_docs_version` in place of "1.9"
also please limit lines to 119 chars